### PR TITLE
feat(start): added force flag

### LINKED ISSUE #148: blocked by "Hoodie is already running!" after server crash
My server crashed badly, I had to destroy the VM and recreate it from the same disk. I guess this is an unusual way for Hoodie to be stopped.
Once I restored the VM, I tried to `hoodie start` but I got the "Hoodie is already running!" message. I discovered that I had to delete `data/hoodie.pid` to be able to successfully restart hoodie.

Maybe you could implement a `-f` option to avoid this kind of problem?

### PR DESCRIPTION
**Pull request by christophwitzko**
Hi,

this should fix #148.

best,
Christoph

### CODE REVIEW

**Comment by janl** [comment]
Oooh nice, wanna add a test for it, too? :)

**Comment by christophwitzko** [comment]
added tests and docs. :+1:

File: spec/cli/start.spec.js [hunk 1]
```diff
@@ -128,4 +133,20 @@ describe('hoodie start', function() {
...
+      var args = {
+        noBrowser: undefined,
+        www: undefined,
+        sudo: undefined,
+        'custom-ports': undefined,
+        verbose: undefined,
+        force: true
+      };
+      cli.argv({ _: ['start'], force: true });
+      expect(hoodie.start.args[0][0]).to.eql(args);
```
Comment (janl) [inline spec/cli/start.spec.js]: we might wanna add a case here where we write a pid file in the test first to simulate a crashed hoodie-server and then make sure we still start.

**Comment by janl** [comment]
Great work overall, though! :)

File: spec/cli/start.spec.js (hunk 1 above)
Comment (christophwitzko) [inline spec/cli/start.spec.js]: :+1: I added some basic integration tests. 6ff2475

**Comment by janl** [comment]
Perfect, thank you! <3 :)

**Comment by christophwitzko** [comment]
:+1: :heart: :relaxed:

**Pull request merged**